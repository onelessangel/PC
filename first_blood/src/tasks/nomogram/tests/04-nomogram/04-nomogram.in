16
19 19
5 1 2 4 1 1
6 1 4 5 1 1 1
7 1 4 2 1 1 1 2
6 2 1 1 2 1 2
6 3 2 2 2 1 2
5 2 1 1 3 1
5 1 1 2 4 1
7 4 1 1 2 1 2 1
5 1 1 2 3 2
6 1 1 3 1 1 1
4 1 1 1 1
6 1 3 2 1 1 1
4 1 5 2 1
7 1 2 1 1 3 1 2
3 5 2 3
4 4 3 3 2
5 3 4 2 2 1
6 1 2 2 2 1 1
5 1 1 2 1 2
4 4 2 3 4
3 3 1 3
6 3 2 1 1 1 5
4 3 3 6 2
4 3 1 2 1
4 3 1 2 5
4 2 2 1 2
5 2 2 3 2 3
5 4 2 1 1 2
5 2 2 1 1 1
7 3 1 2 1 1 3 1
5 2 2 1 2 4
5 1 2 1 2 3
3 2 5 1
3 1 1 5
5 2 2 2 1 4
3 2 1 1
6 5 1 1 2 1 1
4 3 4 1 2
0 0 1 0 1 1 0 0 1 1 1 1 0 1 0 1 0 0 0
1 0 1 1 1 1 0 1 1 1 1 1 0 1 0 1 0 1 0
1 0 1 1 1 1 0 1 1 0 1 0 1 0 1 0 0 1 1
1 1 0 1 0 0 1 0 1 1 0 0 0 0 0 1 0 1 1
1 1 1 0 0 0 1 1 0 1 1 0 1 1 0 1 0 1 1
0 1 1 0 0 1 0 1 0 0 0 1 1 1 0 0 0 1 0
1 0 0 1 0 0 0 0 0 0 1 1 0 1 1 1 1 0 1
1 1 1 1 0 1 0 1 0 1 1 0 0 1 0 1 1 0 1
0 0 0 1 0 1 0 1 1 0 0 1 1 1 0 0 0 1 1
0 0 1 0 1 0 1 1 1 0 1 0 0 0 0 0 1 0 1
0 0 0 1 0 0 1 0 0 1 0 0 0 0 0 0 0 1 0
1 0 1 1 1 0 0 1 1 0 1 0 0 1 0 1 0 0 0
1 0 0 1 1 1 1 1 0 0 0 1 1 0 0 0 0 1 0
1 0 1 1 0 1 0 0 1 0 1 1 1 0 0 1 0 1 1
0 1 1 1 1 1 0 0 0 1 1 0 0 0 1 1 1 0 0
1 1 1 1 0 1 1 1 0 0 1 1 1 0 1 1 0 0 0
1 1 1 0 0 1 1 1 1 0 0 1 1 0 1 1 0 1 0
1 0 1 1 0 0 0 1 1 0 0 1 1 0 1 0 0 0 1
1 0 0 1 0 0 0 0 0 0 1 1 0 0 1 0 0 1 1
19 19
4 2 2 3 1
3 5 1 1
3 1 1 1
4 3 1 2 2
5 2 2 1 3 1
6 1 1 2 2 2 1
6 1 1 2 3 2 2
6 1 4 1 1 4 1
4 3 2 1 2
3 5 4 3
4 5 1 1 1
6 1 1 2 1 2 1
3 3 5 1
5 1 1 1 1 1
7 1 1 2 1 1 1 1
4 1 1 4 1
4 4 1 2 4
5 4 1 1 1 7
5 1 1 1 2 1
6 1 1 1 2 1 1
6 1 2 1 2 3 2
6 1 2 1 2 1 1
4 5 2 1 2
3 1 2 2
3 3 2 2
4 3 1 2 1
4 1 3 1 2
2 1 5
7 1 4 1 2 1 1 1
5 2 2 1 1 1
5 2 1 1 5 1
4 6 2 1 3
5 1 3 1 1 4
7 1 1 4 1 1 1 1
4 2 2 1 3
4 1 2 1 5
7 1 1 1 1 1 1 2
4 1 4 2 3
0 1 1 0 0 0 0 0 0 0 0 1 1 0 1 1 1 0 1
0 0 0 0 0 0 0 0 0 1 1 1 1 1 0 1 0 1 0
0 1 0 0 0 0 0 0 0 0 1 0 1 0 0 0 0 0 0
0 1 1 1 0 0 0 0 0 1 0 0 1 1 0 0 1 1 0
0 0 1 1 0 0 1 1 0 1 0 0 1 1 1 0 1 0 0
1 0 0 1 0 1 1 0 1 1 0 0 1 1 0 0 0 0 1
0 1 0 1 0 1 1 0 0 1 1 1 0 0 1 1 0 1 1
1 0 1 1 1 1 0 1 0 0 1 0 0 1 1 1 1 0 1
0 0 0 0 0 0 1 1 1 0 0 1 1 0 1 0 0 1 1
1 1 1 1 1 0 0 1 1 1 1 0 1 1 1 0 0 0 0
0 1 1 1 1 1 0 0 1 0 0 1 0 0 0 0 0 1 0
1 0 0 0 0 1 0 0 1 1 0 1 0 0 1 1 0 0 1
1 1 1 0 0 0 0 0 1 1 1 1 1 0 0 0 0 0 1
0 1 0 0 0 0 1 0 0 0 0 1 0 0 1 0 0 1 0
0 1 0 1 0 0 1 1 0 1 0 1 0 1 0 0 1 0 0
1 0 0 0 1 0 0 0 0 0 0 0 0 1 1 1 1 0 1
0 0 0 1 1 1 1 0 0 1 0 0 1 1 0 1 1 1 1
1 1 1 1 0 1 0 1 0 0 1 0 1 1 1 1 1 1 1
0 1 0 0 0 0 0 1 0 1 0 1 1 0 0 0 1 0 0
11 17
5 1 2 2 1 2
4 2 1 2 1
6 1 1 1 1 1 2
3 4 4 1
5 1 2 1 1 2
3 2 2 5
3 1 4 4
5 1 2 1 1 3
3 2 4 2
3 1 1 5
2 2 2
1 2
2 2 1
2 5 2
3 1 1 1
4 1 2 1 2
4 2 2 1 1
2 1 5
3 1 2 1
2 3 1
4 1 1 2 1
2 5 1
3 2 2 1
3 1 4 1
4 1 1 1 1
4 3 1 1 1
3 1 2 4
2 1 1
1 0 0 0 0 1 1 1 0 0 1 0 0 1 1 1 1
0 0 1 1 0 1 1 1 0 0 1 1 1 1 1 0 1
0 1 0 0 0 1 1 1 0 0 1 0 0 0 1 1 1
1 1 0 0 0 0 0 1 1 0 0 1 0 0 1 1 0
1 1 0 1 0 1 1 0 0 0 0 1 0 0 1 0 1
0 0 1 0 0 0 0 1 1 1 0 1 1 1 1 0 1
0 1 0 0 1 0 0 0 1 0 1 1 1 0 1 0 1
1 0 1 1 1 0 1 0 1 1 0 0 0 0 1 1 0
0 0 1 1 0 0 0 1 0 1 1 0 0 0 0 0 0
0 0 1 1 0 1 1 0 1 0 0 0 1 0 0 1 0
1 0 0 1 0 0 0 0 1 0 1 1 0 0 0 0 1
15 16
4 1 1 2 2
3 2 2 1
6 3 1 1 3 1 1
2 4 6
4 1 2 3 1
3 2 1 1
3 4 1 1
3 3 1 2
4 1 1 4 1
3 3 2 2
4 1 3 2 3
4 1 2 1 4
3 3 8 2
5 1 1 2 1 2
4 3 1 1 1
4 1 3 3 1
5 1 2 2 1 3
3 4 7 1
5 1 2 2 3 1
4 2 1 1 1
3 1 1 1
4 1 1 2 2
4 1 2 3 3
4 2 2 1 1
3 3 1 3
3 1 2 6
4 1 2 2 1
4 2 1 2 1
3 2 1 3
3 1 2 2
5 1 1 1 1 1
1 0 1 0 0 1 0 1 0 0 0 0 0 0 0 0
0 1 1 1 0 0 0 0 1 0 1 0 1 1 0 1
1 1 0 0 0 0 0 1 0 0 1 0 0 1 1 0
1 0 0 1 0 0 1 0 0 1 0 1 1 1 0 1
1 0 0 0 0 1 1 0 0 1 0 1 0 1 1 1
1 0 0 1 0 1 1 1 0 0 0 0 0 0 1 1
1 1 1 1 1 1 0 1 1 0 0 1 1 0 1 1
1 0 0 0 0 0 1 0 0 1 0 1 0 0 0 0
0 1 1 0 1 0 0 1 0 1 0 0 1 1 0 0
1 1 1 1 1 0 0 1 1 1 0 1 1 0 0 0
1 0 0 1 1 0 0 1 0 1 0 0 0 0 0 0
1 1 0 0 0 1 1 1 1 1 1 1 0 1 0 0
1 0 0 0 1 1 0 0 0 1 0 1 1 0 1 0
0 1 1 1 1 1 1 0 0 1 0 0 1 1 1 0
0 0 0 1 0 1 1 0 0 1 1 0 1 1 1 0
20 10
2 2 4
3 2 1 1
1 7
3 1 2 1
2 6 1
3 1 3 3
3 2 1 1
2 1 5
2 2 1
4 1 1 2 1
1 1
4 1 1 1 1
3 1 3 1
4 2 1 1 1
2 6 1
2 3 2
2 2 1
3 1 1 1
4 1 1 3 1
3 2 1 1
4 2 3 2 2
4 2 1 1 2
5 6 3 2 3 1
4 1 2 3 1
4 6 1 3 2
5 3 3 1 1 1
7 1 1 2 1 2 1 2
6 1 1 4 2 1 1
7 1 1 1 1 1 1 1
6 2 2 1 3 2 1
1 0 1 0 1 1 1 0 0 0
0 0 1 1 0 1 1 1 1 0
1 0 1 1 1 1 1 0 0 1
1 1 0 1 1 1 1 1 1 0
1 1 1 0 1 0 1 0 1 1
0 1 1 1 0 1 1 1 1 0
1 0 0 1 0 1 1 0 1 1
1 1 0 0 0 1 0 1 1 0
1 0 1 1 1 0 1 0 0 1
0 1 1 0 0 0 0 1 0 0
0 0 0 1 1 1 0 0 0 0
1 1 1 1 0 1 1 1 0 1
0 1 0 0 1 1 1 1 0 0
0 1 1 1 0 0 0 1 0 1
0 1 0 0 1 1 0 1 1 0
1 1 0 1 0 1 1 0 0 1
0 0 1 0 0 0 0 1 1 1
1 1 1 1 1 1 0 0 1 1
1 0 0 0 1 0 1 1 1 0
0 0 1 0 1 1 0 1 1 0
12 15
4 1 1 4 1
5 1 2 2 1 1
3 3 1 1
4 2 1 5 3
5 3 3 1 1 1
3 1 7 1
3 1 3 2
4 1 2 1 1
4 1 1 3 1
2 6 4
4 1 1 1 3
3 1 3 6
3 4 1 1
3 1 3 3
4 2 2 1 1
5 1 2 3 1 1
3 1 3 4
3 1 5 1
2 3 1
3 2 1 1
4 2 3 1 1
4 1 1 2 3
4 2 1 2 1
2 2 4
3 2 2 2
2 1 2
3 3 2 1
0 1 0 1 0 1 1 1 0 0 0 1 0 1 0
1 0 0 1 1 1 0 0 1 0 0 0 0 0 0
0 1 1 1 1 1 1 0 1 0 0 0 1 1 1
0 0 0 1 1 0 1 1 1 0 0 0 1 0 0
1 0 1 1 0 1 0 1 1 1 1 0 0 1 0
1 1 1 1 0 1 0 0 0 1 1 1 1 1 0
0 1 0 0 1 0 0 0 1 1 1 0 1 1 1
0 0 0 1 0 0 0 1 1 0 0 1 0 1 1
1 1 0 0 0 1 0 0 0 0 1 0 1 1 1
0 0 0 0 1 0 1 1 1 1 1 0 1 0 1
1 0 1 0 0 0 1 0 0 0 0 0 0 1 0
1 0 1 1 0 1 1 0 0 0 0 1 0 1 1
15 16
4 7 1 2 2
5 3 1 1 2 1
3 5 1 1
3 4 1 3
6 3 1 1 1 1 2
5 2 1 1 1 1
5 1 1 3 3 1
6 2 1 2 1 1 1
4 6 1 1 2
4 1 3 1 3
4 3 1 3 1
4 1 1 5 1
5 1 3 2 3 1
4 1 2 2 5
4 4 1 1 1
3 6 2 1
3 9 1 3
4 5 2 1 1
6 1 2 1 1 2 1
4 1 1 3 2
4 2 1 2 3
2 1 4
3 4 1 3
3 3 1 5
3 2 2 1
3 2 1 2
5 3 1 1 1 5
3 2 2 2
5 1 2 2 2 2
5 2 2 2 1 1
4 1 4 1 2
1 1 1 1 1 1 1 0 1 0 1 1 0 1 1 0
1 1 1 0 0 1 0 0 1 0 1 1 0 0 1 0
1 1 1 1 1 0 0 0 1 0 0 1 0 0 0 0
1 1 1 1 0 0 0 0 0 1 0 0 0 1 1 1
1 1 1 0 0 1 0 1 0 1 0 1 0 1 1 0
1 1 0 1 0 0 0 1 0 0 0 0 1 0 0 1
0 1 0 0 1 0 1 1 1 0 0 1 1 1 0 1
0 1 1 0 1 0 1 1 0 1 0 0 0 1 0 1
0 1 1 1 1 1 1 0 0 1 0 1 0 0 1 1
1 0 0 0 0 1 1 1 0 0 1 0 1 1 1 0
1 1 1 0 0 0 0 0 1 0 0 1 1 1 0 1
0 0 0 1 0 1 0 1 1 1 1 1 0 0 1 0
0 1 0 1 1 1 0 1 1 0 1 1 1 0 0 1
0 1 0 0 1 1 0 1 1 0 0 1 1 1 1 1
1 1 1 1 0 0 0 0 1 0 0 1 0 1 0 0
13 10
3 1 2 2
3 2 1 1
3 1 3 1
2 5 1
3 1 2 1
3 1 5 1
3 3 2 1
3 2 1 1
3 1 1 3
1 1
1 2
2 3 1
3 1 1 3
4 1 1 1 2
4 1 2 3 1
4 3 2 1 1
2 1 4
2 1 4
3 3 4 1
3 1 2 1
4 1 1 1 1
5 3 1 1 1 1
5 1 1 2 1 1
1 0 0 1 1 0 0 1 1 0
0 1 1 0 0 1 0 0 1 0
0 0 1 0 1 1 1 0 1 0
0 1 1 1 1 1 0 0 0 1
0 1 0 1 1 0 0 1 0 0
1 0 1 1 1 1 1 0 0 1
0 1 1 1 0 1 1 0 1 0
1 1 0 0 0 1 0 0 0 1
0 1 0 0 0 1 0 1 1 1
0 0 1 0 0 0 0 0 0 0
0 0 0 0 0 0 0 0 1 1
1 1 1 0 0 0 1 0 0 0
1 0 0 0 0 1 0 1 1 1
16 11
3 3 1 3
2 4 1
3 2 3 3
4 2 2 1 2
3 5 1 1
3 1 1 1
3 1 1 1
3 1 1 1
3 2 2 3
2 1 3
5 2 2 1 1 1
2 2 8
3 1 1 2
3 3 1 4
4 1 2 1 1
3 1 3 2
4 4 2 1 1
5 1 3 1 2 2
5 1 1 1 1 2
5 1 2 2 2 1
4 5 1 2 2
4 3 2 3 1
3 2 2 2
4 2 1 1 3
5 1 1 1 6 1
6 1 2 1 2 1 3
3 4 6 1
0 1 1 1 0 1 0 1 1 1 0
0 0 0 0 1 1 1 1 0 0 1
0 1 1 0 1 1 1 0 1 1 1
1 1 0 1 1 0 0 1 0 1 1
1 1 1 1 1 0 0 0 1 0 1
1 0 0 0 1 0 0 0 0 1 0
1 0 0 1 0 0 0 0 0 0 1
0 0 0 1 0 0 0 1 0 0 1
0 1 1 0 1 1 0 0 1 1 1
0 0 0 0 0 1 0 0 1 1 1
1 1 0 1 1 0 1 0 1 0 1
1 1 0 1 1 1 1 1 1 1 1
0 0 1 0 0 1 0 1 1 0 0
1 1 1 0 0 1 0 1 1 1 1
0 1 0 1 1 0 1 0 0 1 0
1 0 0 0 1 1 1 0 1 1 0
12 14
2 8 4
3 2 4 2
4 1 2 1 2
5 1 2 1 1 1
4 1 2 2 1
3 4 1 2
4 1 1 2 1
4 1 1 1 3
2 6 5
5 1 2 1 1 2
5 1 2 1 1 1
3 3 2 1
2 3 4
3 2 2 3
5 1 1 1 1 1
5 1 1 2 2 1
3 5 3 1
5 2 1 2 1 1
3 2 1 1
3 2 2 1
3 2 1 1
4 2 1 4 2
2 3 4
3 1 1 2
3 1 1 2
3 3 1 3
1 1 1 1 1 1 1 1 0 1 1 1 1 0
1 1 0 0 1 1 1 1 0 1 1 0 0 0
1 0 0 1 1 0 0 0 0 0 1 0 1 1
0 0 1 0 1 1 0 1 0 1 0 0 0 1
0 1 0 1 1 0 0 1 1 0 0 0 0 1
1 1 1 1 0 1 0 0 1 1 0 0 0 0
1 0 0 0 0 1 0 0 0 1 1 0 0 1
1 0 0 0 1 0 1 0 1 1 1 0 0 0
1 1 1 1 1 1 0 0 0 1 1 1 1 1
0 1 0 1 1 0 0 1 0 0 1 0 1 1
0 1 0 0 0 1 1 0 0 1 0 1 0 1
0 0 1 1 1 0 0 0 1 1 0 1 0 0
14 11
4 1 2 1 1
4 1 1 1 1
4 4 1 1 1
2 3 2
3 1 3 1
3 1 3 1
1 3
3 3 4 1
4 1 1 1 1
3 2 3 2
3 1 2 1
3 1 1 3
4 2 1 1 1
3 3 1 1
4 2 1 1 1
4 2 2 1 1
4 1 2 5 2
3 5 2 1
3 1 1 1
6 1 1 1 1 1 3
3 1 1 3
3 4 1 4
5 1 1 1 1 2
4 2 1 2 3
3 1 1 2
0 1 0 0 1 0 0 0 1 0 1
0 1 1 1 1 0 1 1 1 1 0
0 0 1 1 1 0 0 1 0 1 0
1 1 0 1 1 0 1 1 0 0 1
0 0 0 1 0 1 0 0 1 1 0
1 1 0 0 0 0 1 1 1 0 0
1 1 1 1 0 0 1 1 0 0 1
0 0 1 1 1 0 1 0 0 1 0
1 0 0 0 1 0 0 1 0 1 0
0 1 0 1 0 1 1 0 1 0 1
1 0 1 0 1 1 0 0 0 1 1
1 1 0 1 0 1 0 0 0 1 1
1 1 1 0 0 1 0 1 0 1 0
0 0 0 0 0 0 0 1 0 0 1
20 17
2 1 3
3 1 1 1
5 2 1 2 3 1
5 4 1 1 1 2
3 1 5 1
5 3 1 2 1 1
5 2 1 2 1 2
4 1 3 1 1
3 1 1 5
3 2 4 3
5 1 2 3 1 4
6 1 1 2 1 5 1
4 4 1 1 1
3 2 2 1
4 1 2 3 4
5 3 1 2 2 3
5 2 1 2 3 1
4 2 2 1 2
6 2 1 1 1 2 2
3 2 1 2
6 1 1 1 2 1 2
4 4 1 2 2
4 2 1 2 6
6 2 1 2 2 2 1
7 1 1 1 2 1 1 1
3 2 1 5
5 1 1 2 1 1
6 2 2 2 1 1 1
5 2 1 4 3 1
6 1 1 1 2 1 1
5 1 2 2 5 1
4 2 4 1 2
4 1 2 1 2
4 3 2 3 3
5 1 1 1 3 3
7 1 1 2 2 1 2 2
5 4 1 2 1 3
0 1 1 0 1 0 1 0 1 1 0 1 1 0 0 1 1
1 0 0 1 0 0 1 1 0 1 1 0 0 0 1 0 1
1 0 0 0 1 0 0 0 0 0 0 1 1 0 0 1 1
0 0 0 1 0 1 1 1 1 0 0 1 0 0 1 1 0
0 1 0 1 0 0 1 0 1 1 0 0 1 1 0 1 1
1 1 0 0 0 0 1 0 1 1 1 1 1 1 1 1 1
1 1 0 0 1 1 0 1 1 1 0 0 1 0 1 1 0
1 0 0 1 0 1 0 1 1 1 0 1 0 1 1 0 0
1 1 0 1 0 1 0 0 1 0 1 1 1 1 1 0 0
0 0 1 0 1 0 0 0 0 0 0 0 0 0 0 0 1
1 1 0 0 0 1 0 0 1 1 1 0 1 1 1 1 0
1 1 1 0 1 1 0 0 1 1 1 1 1 1 0 1 1
0 0 0 0 0 0 1 0 1 0 0 1 1 1 1 1 0
0 1 0 1 0 1 1 1 0 0 0 0 0 1 0 1 1
1 1 1 0 0 1 1 0 1 1 1 1 0 0 0 0 1
0 1 0 1 0 0 1 1 1 0 0 0 0 0 1 1 0
1 1 0 1 1 1 1 0 1 1 0 1 1 0 0 0 0
1 0 0 0 1 1 0 0 1 0 0 1 0 1 1 1 0
0 0 1 1 1 1 0 1 0 0 0 0 0 1 1 0 0
0 0 0 0 1 0 1 0 1 0 1 0 1 0 0 0 0
12 20
5 4 2 1 1 1
4 2 1 1 1
6 3 3 1 1 2 1
6 2 1 3 3 2 1
4 1 7 1 2
7 3 1 1 2 1 1 2
5 1 1 3 2 2
5 4 2 1 1 2
8 1 1 2 1 1 1 1 3
5 1 3 3 2 1
6 3 2 3 1 1 1
4 2 5 1 1
3 2 1 2
4 1 4 1 1
4 1 1 4 1
4 2 1 1 1
3 3 2 4
4 1 1 2 1
3 4 1 2
4 1 2 2 1
3 7 1 1
2 3 2
4 1 2 1 3
4 1 3 1 4
5 1 1 2 1 1
3 1 2 1
1 3
4 1 1 2 1
2 3 2
4 1 1 1 2
3 2 1 2
3 1 2 2
1 0 0 1 0 0 1 1 0 0 0 1 0 0 0 0 1 1 1 1
0 1 1 1 1 1 1 0 0 0 1 1 0 1 1 1 0 0 1 0
0 0 1 1 0 1 0 0 0 0 1 0 0 0 1 0 1 1 1 0
1 0 0 0 1 1 1 1 0 1 1 1 1 1 1 0 1 1 1 1
0 1 0 0 0 0 1 1 1 0 0 1 0 1 0 0 1 1 0 0
0 1 1 1 0 0 0 0 1 1 0 1 0 0 1 1 0 1 1 0
1 0 1 1 1 0 1 0 1 1 1 1 1 1 0 1 0 1 0 0
0 1 1 1 1 1 0 1 1 1 0 1 1 0 1 1 1 1 0 0
0 0 1 1 0 0 1 0 1 0 1 0 0 0 1 1 1 0 0 0
0 0 1 0 0 1 0 0 1 1 1 1 0 1 0 0 1 1 0 1
0 1 1 1 0 0 0 1 0 1 0 1 1 1 0 1 0 1 0 1
0 1 0 1 0 0 0 0 1 1 1 0 1 0 0 0 0 0 1 0
13 10
3 1 3 2
2 3 5
3 1 2 1
3 1 3 2
4 1 2 1 1
3 1 1 1
2 1 1
4 2 1 1 3
3 4 1 2
3 1 1 1
4 1 4 1 1
2 1 4
3 1 1 1
3 4 4 1
3 1 1 2
4 4 1 1 3
3 3 2 1
5 2 2 1 1 1
3 2 1 4
3 2 2 1
4 1 1 1 3
3 3 2 1
4 1 2 2 2
0 1 1 0 1 1 0 0 1 0
1 0 0 1 0 1 1 1 1 0
1 0 0 1 0 1 0 1 0 1
0 1 0 0 0 0 1 0 0 1
0 1 1 1 1 0 0 0 0 1
1 0 0 0 0 0 0 1 0 0
1 0 0 0 0 0 1 1 1 1
0 1 1 1 0 1 0 0 1 1
0 1 0 1 1 0 1 1 0 1
0 1 0 0 1 0 0 1 1 0
0 1 1 1 1 0 1 0 1 0
0 0 0 0 1 1 1 1 1 1
1 0 0 1 0 1 0 1 1 0
14 10
2 1 1
1 2
3 2 1 2
3 2 4 1
4 2 3 1 1
3 4 1 1
3 3 2 1
0
2 4 2
2 6 1
4 2 1 1 1
3 2 2 2
3 1 2 5
3 1 1 1
4 1 2 2 1
2 5 4
3 1 2 5
4 1 3 2 2
2 2 3
5 1 2 1 1 3
5 1 2 1 1 1
3 4 1 2
3 2 1 2
3 1 3 4
0 0 0 0 0 1 0 0 0 1
0 0 0 0 0 0 0 1 1 0
1 1 0 1 0 0 0 1 1 0
0 1 1 0 1 1 1 1 0 1
1 1 0 1 1 1 0 1 0 1
1 1 1 1 0 0 1 0 0 1
0 1 1 1 0 1 1 0 1 0
0 0 0 0 0 0 0 0 0 0
1 1 1 1 0 0 1 1 0 0
1 1 1 1 1 1 0 0 0 1
0 1 1 0 1 0 1 0 0 1
0 1 1 0 1 1 0 0 1 1
1 0 1 1 0 1 1 1 1 1
0 0 0 1 0 1 0 1 0 0
17 15
6 1 1 1 1 2 1
3 4 1 1
4 1 3 2 1
4 1 3 3 1
3 1 5 6
3 2 2 4
3 4 2 3
4 1 2 2 1
4 2 5 1 2
4 1 3 3 4
5 1 2 2 1 1
3 1 2 1
3 4 1 2
3 2 2 1
4 5 2 2 1
4 1 1 1 1
2 4 3
4 1 1 1 1
6 1 1 3 1 1 1
7 1 1 1 2 1 1 1
6 5 2 2 1 1 1
4 6 3 1 3
5 3 2 2 1 1
3 1 4 1
4 1 1 4 3
5 1 1 1 2 4
3 1 4 1
4 1 4 3 2
5 2 4 2 3 1
4 3 1 2 1
5 2 1 2 2 1
6 1 1 1 2 1 1
1 1 0 0 1 0 0 1 0 1 0 1 1 1 1
0 1 0 0 1 0 0 1 0 0 0 1 1 0 1
1 1 1 0 1 1 1 0 1 1 0 1 0 0 1
1 0 1 1 0 1 1 0 0 1 1 1 1 1 1
0 0 1 1 1 0 1 0 1 0 1 0 1 0 0
1 1 0 1 0 0 0 0 1 0 1 0 0 0 0
1 0 1 1 0 1 1 1 0 0 1 0 0 0 1
0 1 1 0 0 1 1 0 1 1 0 0 0 0 0
0 0 1 0 0 1 0 0 0 1 0 0 0 1 0
0 1 0 0 1 0 0 0 1 0 1 1 0 0 0
0 1 0 1 0 0 1 1 1 0 0 1 0 1 1
1 1 0 0 0 0 0 0 1 1 1 1 0 0 1
0 0 0 1 0 0 1 1 1 0 1 0 0 1 1
0 1 1 0 1 1 1 0 0 0 1 1 1 0 0
1 0 0 0 1 1 0 0 0 1 0 1 1 0 0
0 1 1 1 1 1 0 0 1 1 1 0 1 1 0
1 1 0 1 1 0 1 1 0 1 1 1 1 1 1
