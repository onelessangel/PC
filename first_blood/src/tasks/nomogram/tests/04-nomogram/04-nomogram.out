Corect
Corect
Eroare
Eroare
Eroare
Eroare
Corect
Corect
Corect
Corect
Eroare
Eroare
Eroare
Eroare
Corect
Eroare
